Tests/LibWeb/Text/expected/all-window-properties.txt

```diff
@@ -419,6 +420,7 @@ TransformStream
 TransformStreamDefaultController
 TransitionEvent
 TreeWalker
+TrustedTypePolicyFactory
```

Comment:
This property is added to this text file in the first commit, but it should be added later, in the commit that defines this interface.